en/sql/tuning.rst

```diff
@@ -3829,6 +3829,89 @@ The join with the *right_tbl* table was eliminated.
                 9           60          600  'Left- 806'                     6
                10           60          600  'Left- 906'                     6
 
+.. _pred-push:
+
+Predicate Push
+-----------------------
+
+**Predicate Push** is an optimization that pushes the predicate of the main query into the view.
+
+By applying **Predicate Push** to the subquery, the amount of data queried in the main query is reduced.
+
+For instance, when performing a join using the join condition *a.code = r.athlete_code* as in the query below, if the external predicate can be pushed into the query block, the amount of data to be joined can be reduced.
+
+.. code-block:: sql
+
+        SELECT a.name, r.score 
+        FROM (SELECT name, nation_code, code, count(*) cnt FROM athlete GROUP BY name, nation_code) a, record r
+        WHERE a.code = r.athlete_code
+        AND a.nation_code = 'KOR';
+
+There are no predicates in the inline-view of the query above. If the query transformation doesn't work, 
+the *athlete* table would have been fully scanned and joined before filtering with the condition *a.nation_code = 'KOR'*.
+
+However, with **Predicate Push**, if the query is transformed as follows, the optimization ensures that only fewer amounts of data are queried.
+
+.. code-block:: sql
+
+        SELECT a.name, r.score 
+        FROM (SELECT name, nation_code, code, count(*) cnt FROM athlete WHERE nation_code = 'KOR' GROUP BY name, nation_code ) a, record r
+        WHERE a.code = r.athlete_code;
+
+In CUBRID, **Predicate Push** can't be performed if the query meets the following conditions:
+
+#. Using **NO_PUSH_PRED** hint on main query.
+
+#. Contains **CONNECT BY**.
+    
+#. Using aggregation or analytical functions.
+
+#. Using **ROWNUM, LIMIT**, or **GROUPBY_NUM(), INST_NUM(), ORDERBY_NUM()**.
+    
+#. Written using **Correlated Subquery**.
+
+#. When subqueries are used in predicates.
+
+#. The view includes methods.
```

Comment:
Is it necessary to mention the view here when it hasn't been mentioned consistently from the start?
How about below?
```
Using **methed**.
```